packages/docs/src/locales/en/en.json

```diff
@@ -1134,12 +1134,15 @@
     },
     "VaTimePicker": {
```

Comment:
`AM` \ `PM` toggle changes selection to invalid time.

![amPmToggleSelectsInvalidTime](https://user-images.githubusercontent.com/7745899/170986188-d437ac32-c55d-444d-84bf-5c6510caa9c4.gif)

Reply:
Still relevant